tests/e2e/specs/menu.cy.js

```diff
@@ -76,6 +76,11 @@ describe('CylcObject Menu component', () => {
       .then(($el) => {
         firstID = $el.text().trim()
       })
+      // Expand menu & check for the presence of edit-runtime at family level:
+      .get('#less-more-button')
+      .click()
+      .get('.c-mutation')
+      .contains('Edit Runtime')
```

Comment:
There is a spec for edit runtime here `tests/e2e/specs/editRuntimeForm.cy.js`, I think that would be a better place

Reply:
Thank you -